platformio/src/locales/locale_et_EE.inc

```diff
@@ -0,0 +1,438 @@
+//   the glibc source tree, in files named after the locale codes and encoding
+//   schemes. For example, the en_US locale data is stored in the
+//   localedata/locales/en_US file, which contains the definitions for date and
+//   time formats, month and day names, and other time-related settings for the
+//   LC_TIME category.
+//   https://www.gnu.org/software/libc/sources.html
+// D_T_FMT                 string for formatting date and time
+const char *LC_D_T_FMT     = "%a %d %b %Y %T";
+// LC_D_FMT                date format string
+const char *LC_D_FMT       = "%d.%m.%Y";
+// T_FMT                   time format string
+const char *LC_T_FMT       = "%T";
+// T_FMT_AMPM              a.m. or p.m. time format string
+const char *LC_T_FMT_AMPM  = "%I:%M:%S %p";
+// AM_STR                  Ante Meridian affix
+const char *LC_AM_STR      = "AM";
+// PM_STR                  Post Meridian affix
+const char *LC_PM_STR      = "PM";
+// DAY_{1-7}               name of the n-th day of the week
+const char *LC_DAY[7]      = {"P\xFC""hap\xE4""ev", "Esmasp\xE4""ev", "Teisip\xE4""ev", "Kolmap\xE4""ev",
+                              "Neljap\xE4""ev", "Reede", "Laup\xE4""ev"};
+// ABDAY_{1-7}             abbreviated name of the n-th day of the week
+const char *LC_ABDAY[7]    = {"P", "E", "T", "K", "N", "R", "L"};
+// MON_{1-12}              name of the n-th month of the year
+const char *LC_MON[12]     = {"Jaanuar",    "Veebruar", "M\xE4rts", "Aprill",
+                               "Mai",       "Juuni",    "Juuli",    "August",
+                               "September", "Oktoober", "November", "Detsember"};
+// ABMON_{1-12}            abbreviated name of the n-th month of the year
+const char *LC_ABMON[12]   = {"Jaan", "Veebr", "M\xE4rts", "Apr", "Mai", "Juuni",
+                              "Juuli", "Aug", "Sept", "Okt", "Nov", "Dets"};
+// ERA                     era description segments
+const char *LC_ERA         = "";
+// ERA_D_FMT               era date format string
+const char *LC_ERA_D_FMT   = "";
+// ERA_D_T_FMT             era date and time format string
+const char *LC_ERA_D_T_FMT = "";
+// ERA_T_FMT               era time format string
+const char *LC_ERA_T_FMT   = "";
+
+// OWM LANGUAGE
+// For full list of languages, see
+// https://openweathermap.org/api/one-call-api#multi
+// Note: "[only] The contents of the 'description' field will be translated."
+const String OWM_LANG = "en";
+
+// CURRENT CONDITIONS
+const char *TXT_FEELS_LIKE         = "Tunnetuslik";
+const char *TXT_SUNRISE            = "P\xE4ikeset\xF5us";
+const char *TXT_SUNSET             = "Loojang";
+const char *TXT_WIND               = "Tuul";
+const char *TXT_HUMIDITY           = "\xD5huniiskus";
+const char *TXT_UV_INDEX           = "UV Indeks";
+const char *TXT_PRESSURE           = "R\xF5hk";
+const char *TXT_AIR_QUALITY        = "\xD5hu Kvaliteet";
+const char *TXT_AIR_POLLUTION      = "\xD5husaaste";
```

Comment:
Please make sure the air pollution translation makes sense here (I used google translate haha).

Reply:
It does, all good 👍